membership-guide.md

```diff
@@ -9,29 +9,31 @@ As an MLH Member Event all hackathons receive the following benefits.
 0. **Listed on Season Page.** Events will be listed on their region's official hackathon season page.
 0. **Onsite Support.** MLHers are extensions of MLH Member Event organizing teams. They’re able to help coordinate judging, mentor hackers, set up food, or even take out the trash. Whatever a hackathon needs to ensure that an event runs smoothly MLH will be available to provide assistance.
 0. **Office Hours.** Our team helps support over 200 student hackathons every year. We've helped deal with all sorts of problems and questions. With MLH Office Hours, you can tap into the MLH team's expert advice and mentorship when you need it in a 1-on-1 environment.
+0. **MLH emergency budget** All MLH Member Events encounter a 30-50% attrition rate in registrations. To combat this, MLH Member events typically overbook for the hackathon. The uncertainty of attendance can be scary, so we’ll help you. If your event overbooks and as a result exceeds capacity, we will pay for the food costs of the additional hackers in attendance.
```

Comment:
Can we make this more positive? "If you overshoot your attendance goal, we'll pay for the food costs, to celebrate and also make sure your event doesn't go over budget."